doc/release/v1_4_0.md

```diff
@@ -13,6 +13,11 @@ Important Changes
   compatibility.
 * Added `target_include_directories` in all RTF
   libraries.
+* The RTF libraries library can now be compiled either as shared or 
```

Comment:
If we decide to set cmake_minimum_required to 3.5 we have to put this information in Important changes in the changelog

Reply:
Indeed, adding now.

Reply:
Fixed in https://github.com/robotology/robot-testing/pull/85/commits/25fbf265ec27504c89d172ddadb7407e6393ce1f .